src/main/java/ezra/Ezra.java

```diff
@@ -0,0 +1,61 @@
+package ezra;
+
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+/**
+ * Main class for the Ezra chatbot.
+ */
+public class Ezra {
+
+    private Storage storage;
+    private TaskList tasks;
+
+    /**
+     * Constructs an Ezra object with the specified file path to load saved tasks from.
+     *
+     * @param filepath The file path for storing tasks.
+     */
+    public Ezra(String filepath) {
+        this.storage = new Storage(filepath);
+        try {
+            this.tasks = new TaskList(storage.load());
+        } catch (FileNotFoundException e) {
+            this.tasks = new TaskList();
+        }
+    }
+
+    /**
+     * Runs the Ezra application, allowing the user to enter commands.
+     */
+    public void run() {
+        Ui.greet();
+
+        Scanner scanner = new Scanner(System.in);
+        while (scanner.hasNextLine()) {
+            String input = scanner.nextLine();
+            Parser.generateReply(input, storage, tasks);
+            if (input.equals("bye")) {
+                break;
+            }
+        }
+        scanner.close();
+    }
+
+    /**
+     * You should have your own function to generate a response to user input.
+     * Replace this stub with your completed method.
+     */
```

Comment:
I am a little confused what this method does, and I am not completely understanding what the Javadoc is saying either. Is this something that the user should change? What if they download your release as a .jar file? If this method has been completed, it might be good to revise the Javadoc to give a more informative description of the method, including `@param` and `@return`.

Reply:
Thanks for pointing that out, I will update the javadoc